<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-02-13T16:41:33 IST" name="Default suite" finished-at="2020-02-13T16:42:15 IST" duration-ms="42064">
    <groups>
    </groups>
    <test started-at="2020-02-13T16:41:33 IST" name="Default test" finished-at="2020-02-13T16:42:15 IST" duration-ms="42064">
      <class name="TestCases.ClickOnSignUpTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:TestCases.ClickOnSignUpTest@223f3642]" started-at="2020-02-13T16:41:33 IST" name="setUp" finished-at="2020-02-13T16:41:33 IST" duration-ms="30" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="testOpenApplication()[pri:0, instance:TestCases.ClickOnSignUpTest@223f3642]" started-at="2020-02-13T16:41:33 IST" name="testOpenApplication" finished-at="2020-02-13T16:41:58 IST" duration-ms="24756" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testOpenApplication -->
        <test-method is-config="true" signature="methodLevelSetup()[pri:0, instance:TestCases.ClickOnSignUpTest@223f3642]" started-at="2020-02-13T16:41:58 IST" name="methodLevelSetup" finished-at="2020-02-13T16:41:58 IST" duration-ms="40" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- methodLevelSetup -->
        <test-method signature="testClinkOnLink(java.lang.reflect.Method)[pri:0, instance:TestCases.ClickOnSignUpTest@223f3642]" started-at="2020-02-13T16:41:58 IST" name="testClinkOnLink" description="Testing clicked on Link or not." finished-at="2020-02-13T16:42:15 IST" duration-ms="16806" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestCases.ClickOnSignUpTest.testClinkOnLink(java.lang.reflect.Method) throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testClinkOnLink -->
        <test-method signature="testFailedScenarios(java.lang.reflect.Method)[pri:1, instance:TestCases.ClickOnSignUpTest@223f3642]" started-at="2020-02-13T16:42:15 IST" name="testFailedScenarios" description="Test filed test cases " finished-at="2020-02-13T16:42:15 IST" duration-ms="6" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestCases.ClickOnSignUpTest.testFailedScenarios(java.lang.reflect.Method)]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[did not expect to find [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at TestCases.ClickOnSignUpTest.testFailedScenarios(ClickOnSignUpTest.java:54)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testFailedScenarios -->
        <test-method is-config="true" signature="methodLevelSetup()[pri:0, instance:TestCases.ClickOnSignUpTest@223f3642]" started-at="2020-02-13T16:42:15 IST" name="methodLevelSetup" finished-at="2020-02-13T16:42:15 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- methodLevelSetup -->
        <test-method is-config="true" signature="methodLevelSetup()[pri:0, instance:TestCases.ClickOnSignUpTest@223f3642]" started-at="2020-02-13T16:42:15 IST" name="methodLevelSetup" finished-at="2020-02-13T16:42:15 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- methodLevelSetup -->
        <test-method signature="testAcualExpected(java.lang.reflect.Method)[pri:2, instance:TestCases.ClickOnSignUpTest@223f3642]" started-at="2020-02-13T16:42:15 IST" name="testAcualExpected" description="Test actual expected" finished-at="2020-02-13T16:42:15 IST" duration-ms="2" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void TestCases.ClickOnSignUpTest.testAcualExpected(java.lang.reflect.Method)]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [That] but found [This]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [That] but found [This]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at TestCases.ClickOnSignUpTest.testAcualExpected(ClickOnSignUpTest.java:63)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testAcualExpected -->
        <test-method is-config="true" signature="closeDriver()[pri:0, instance:TestCases.ClickOnSignUpTest@223f3642]" started-at="2020-02-13T16:42:15 IST" name="closeDriver" finished-at="2020-02-13T16:42:15 IST" duration-ms="340" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeDriver -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:TestCases.ClickOnSignUpTest@223f3642]" started-at="2020-02-13T16:42:15 IST" name="tearDown" finished-at="2020-02-13T16:42:16 IST" duration-ms="530" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TestCases.ClickOnSignUpTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
